application/controllers/digital_take_up.py

```diff
@@ -84,7 +131,27 @@ def channel_options(admin_client, uuid):
         if True in form.data.values():
```

Comment:
Sure, that makes sense. I assumed we were checking for the presence of values instead.